lib/CMakeLists.txt

```diff
@@ -29,6 +29,9 @@ set(LinkedLibs
   # Conversion passes.
   TorchMLIRConversionPasses
   TorchMLIRTorchOnnxToTorch
+
+  StablehloPasses
+  StablehloLinalgTransforms
```

Comment:
Can you move these to a `list(APPEND` following that is gated on `TORCH_MLIR_ENABLE_STABLEHLO`?